backend/typescript/models/behvaiourLevelDetails.model.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  Column,
+  Model,
+  Table,
+  ForeignKey,
+  DataType,
+} from "sequelize-typescript";
+import Behaviour from "./behaviour.model";
+import { BehaviourLevel } from "../types";
+
+@Table({ timestamps: false, tableName: "behvaiour_level_details" })
+export default class BehaviourLevelDetails extends Model {
+  @Column({ type: DataType.INTEGER, allowNull: false })
+  id!: number;
+
+  @ForeignKey(() => Behaviour)
+  @Column({ type: DataType.INTEGER, allowNull: false })
+  behaviour_id!: number;
+
+  @Column({ type: DataType.ENUM, allowNull: false })
+  level!: BehaviourLevel;
+
+  @Column({ type: DataType.STRING, allowNull: false })
+  description!: string;
```

Comment:
I think we can let description and training_instructions to be nullable.

Please change this here in the model, and also the migration file. (sorry I didn't specify this in the ticket)